760.find-anagram-mappings.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * @param {number[]} nums1
+ * @param {number[]} nums2
+ * @return {number[]}
+ */
+
+/* URL of this problem
+    https://leetcode.com/problems/find-anagram-mappings/
+*/
+
+var anagramMappings = function(nums1, nums2) {
```

Comment:
Can you use Array.prototype.map() to improve this?

Reply:
I modified the code with array.map().